### DISCUSSION TITLE
how to switch back to official server

### DISCUSSION THREAD

**Discussion by rafroad** [Melledy/Grasscutter#1256] [Q&A]
i was trying to host my own server then i use fiddler to and the script in the wiki now when i try to login to the official server i have a code 4204 error how to fix this?

**Comment by diona-katzlein** [comment]
just clear all proxy, close fiddler and delete proxy script. if you ask how to clear proxy? just find on google

**Reply by rafroad** [reply]
i already did disable fiddler and the proxy but it's still gives me the error

**Reply by rafroad** [reply]
but the weird thing is when i launch from glassclipper x i didn't get the error but when i launch using the official launcher i get the error

**Reply by diona-katzlein** [reply]
Open CMD
Paste this:
reg add "HKCU\Software\Microsoft\Windows\CurrentVersion\Internet Settings" /v ProxyEnable /t REG_DWORD /d "%ORIG_PROXY_ENABLE%" /f >nul 2>nul
reg add "HKCU\Software\Microsoft\Windows\CurrentVersion\Internet Settings" /v ProxyServer /d "%ORIG_PROXY_SERVER%" /f >nul 2>nul

**Reply by diona-katzlein** [reply]
Try open http://127.0.0.1:8080/ on browser, if say 502 Bad Gateway mean proxy still active or try troubleshoot.

**Reply by rafroad** [reply]
google chrome says error connection refused after pasting the command

**Reply by rafroad** [reply]
and the error still pop up in the game

**Comment by diona-katzlein** [comment]
Solved Genshin Impact error 4204!!!! open config file n set channel value = 1 or 0 sub channel, = 0 or 1 depend.. I able to observe u can set -2, -3.... till one digit in both but u cant do that in plus integer like 2, 3 ....(pic r about making this error )

Found on redit: /r/GenshinImpactTips/

**Reply by rafroad** [reply]
still didn't work i set the channel to 1 and sub to 0

**Comment by nitrog0d** [comment]
it brokey

**Reply by rafroad** [reply]
any idea how to fix it?
